src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -457,6 +459,7 @@ def load_arguments(self, _):
         c.argument('enable_vpa', action='store_true', is_preview=True, help="enable vertical pod autoscaler for cluster")
         c.argument('disable_vpa', action='store_true', is_preview=True, help="disable vertical pod autoscaler for cluster")
         c.argument('cluster_snapshot_id', validator=validate_cluster_snapshot_id, is_preview=True)
+        c.argument('custom_ca_trust_certificates', options_list=["--custom_ca_trust_certificates", "--ca-certs"], validator=validate_custom_ca_trust_certificates, is_preview=True)
```

Comment:
```suggestion
        c.argument('custom_ca_trust_certificates', options_list=["--custom-ca-trust-certificates", "--ca-certs"], validator=validate_custom_ca_trust_certificates, is_preview=True)
```

Use `-` instead of `_`.